### DISCUSSION TITLE
Ezra Bible App - Year 2021 in review :)

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#600] [General]
This is the year 2021 in review from an Ezra Bible App perspective!

* We did six releases (going from 0.17.0 to 1.4.0).
* Android was introduced as fourth platform.
* The development team grew from 1-2 people. Yay!
* The translator team grew from 4-7 people and four additional languages were added: Romanian, Russian, Ukrainian and Brazilian-Portuguese. Yay!
* 150 features/enhancements/issues have been resolved.
* The JavaScript code base grew from 12K lines of code to ~20K lines of code.
* Major features added:
   * Added menu to adjust font-size for Bible text.
   * Desktop app: Added emoji picker to input fields.
   * Added functionality for opening and navigating book chapters individually.
   * Dynamically switchable UI translation.
   * Notes document export.
   * Many usability enhancements.

As the maintainer of Ezra Bible App, I am very thankful for what happened in the last year, especially the growth of the team! May this all be to God's glory and to the benefit of our user's Bible study!

Tobias

**Comment by guiquimthe** [comment]
Thank God for the development and translation team. Thanks for the Ezra Bible App software. I've been studying the Bible better through it. I hope to contribute something.

Guilherme
